src/app/pages/admin-page/admin-page-routing.module.ts

```diff
@@ -1,11 +1,28 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { AdminPageComponent } from './admin-page.component';
+import { TableComponent } from './admin-table/table.component';
+import { VendorFormComponent } from './vendor-form/vendor-form.component';
 
 const routes: Routes = [
   {
     path: '',
-    component: AdminPageComponent
+    component: AdminPageComponent,
+    children: [
+      {
+        path: 'vendors',
```

Comment:
maybe we'll do it just ```path: '',``` for now (so that the page with vendors opens immediately, and then we'll change it)

Reply:
it provides another problem